CHANGELOG.md

```diff
@@ -3,30 +3,58 @@
 
 All notable changes to this project will be documented in this file.
 
-The format is based on [Keep a Changelog](http://keepachangelog.com/)
-and this project adheres to [Semantic Versioning](http://semver.org/).
+The format is based on [Keep a Changelog] and this project adheres to [Semantic Versioning].
 
 ## [Unreleased] - yyyy-mm-dd
 
-Initial release of a PyTorch Dataset for SOMA Experiments. This is a port and
-enhancement of code contributed by the Chan Zuckerberg Initiative Foundation
-[CELLxGENE](https://cellxgene.cziscience.com/) project.
+Initial release of a [PyTorch Dataset] for [SOMA] Experiments. This is a port and enhancement of code contributed by the Chan Zuckerberg Initiative Foundation [CELLxGENE] project.
 
-This is not a one-for-one migration of the contributed code. Substantial changes have
-been made to the package utility (e.g., multi-GPU support), improved API UX, performance
-improvements, and more.
+This is not a one-for-one migration of the contributed code. Substantial changes have been made to the package utility (e.g., multi-GPU support), improved API UX, performance improvements, and more.
 
 ### Added
 
-- Initial project organization and other scaffolding [PR #4](https://github.com/single-cell-data/TileDB-SOMA-ML/pull/4)
-- Simple, non-shuffling Dataset/DataPipe implementation [PR #6](https://github.com/single-cell-data/TileDB-SOMA-ML/pull/6)
-- Add CI workflows [PR #7](https://github.com/single-cell-data/TileDB-SOMA-ML/pull/7)
-- Add a DataLoader creation wrapper function [PR #8](https://github.com/single-cell-data/TileDB-SOMA-ML/pull/8)
-- Add shuffling support [PR #9](https://github.com/single-cell-data/TileDB-SOMA-ML/pull/9)
-- Add first draft of tutorial notebooks [PR #10](https://github.com/single-cell-data/TileDB-SOMA-ML/pull/10)
-- Archive script used to populate the repo commit history [PR #11](https://github.com/single-cell-data/TileDB-SOMA-ML/pull/11)
-- I/O buffer performance optimization [PR #13](https://github.com/single-cell-data/TileDB-SOMA-ML/pull/13)
+#### 2025-02
```

Comment:
As a change log, this is burying the lede -- the public API changed.  I would summarize the change for ease of use by a person using the API (e.g., "ExperimentIterableDataPipe is removed. ExperimentIerableDataset is renamed ExperimentDataset. ...")